src/vip_ipykernel/__init__.py

```diff
@@ -0,0 +1,28 @@
+__version__ = '0.1.0a3'
+
+import sys
+from pathlib import Path
+
+VENV_NAMES = [
+    '.venv',
+    'venv',
+]
+
+ANCHOR = Path(Path.cwd().anchor)
+
+
+def venv_search(prefix: Path = Path('.')) -> Path:
+    prefix = prefix.absolute()
+
+    found_venvs = []
+
+    for venv in VENV_NAMES:
+        found_venvs.extend(prefix.glob(f"{venv}/bin/python3"))
```

Comment:
It doesn't look like there's any actual glob pattern involved? Might be clearer to just check if the exact path exists:

```python
path = prefix / venv / 'bin' / 'python3'
if path.is_file():
    return path.absolute()
```

Reply:
Ah yes good point, originally I had a reason to use glob which escapes me now so I'll change it, thanks